src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -353,4 +364,8 @@
     </PropertyGroup>
     <Error Condition="!Exists('..\packages\xunit.runner.visualstudio.2.1.0\build\net20\xunit.runner.visualstudio.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\xunit.runner.visualstudio.2.1.0\build\net20\xunit.runner.visualstudio.props'))" />
   </Target>
+  <Target Name="CopySchema" AfterTargets="Build">
+    <Copy SourceFiles="..\Sarif\Autogenerated\Sarif.schema.json"
+          DestinationFolder="$(OutDir)" />
+  </Target>
```

Comment:
You will be more resilient to project changes if you root your copy against the $(SolutionDir) variable rather than using relative paths to retrieve specific files. I think $(SolutionDir)Sarif\Autogenerated\Sarif.schema.json should work. Note that VS vars that refer to directories are suffixed with a trailing slash.